packages/integration-tests/.eslintrc.json

```diff
@@ -1,10 +1,21 @@
 {
   "extends": ["plugin:cypress/recommended", "../../.eslintrc.json"],
-  "ignorePatterns": ["!**/*"],
+  "ignorePatterns": ["!**/*", "node_modules/*"],
```

Comment:
Then it must be something having to do with `nx`, because linting with the above configuration takes about 8 secons, while if not excluding `node_modules`, I've waited longer than 1 minute before cancelling the process.

So I'd keep it.

Reply:
well `"!**/*"` it is telling to not ignore anything... why is it there?

Reply:
Looks like it may have to do with the configuration they use in the "lint" action. If removed, it fails
![lint-patterns](https://user-images.githubusercontent.com/829045/182112155-cd20c20b-0da1-42c5-ae4f-79323c0f6204.png)